sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/AuthClient.java

```diff
@@ -0,0 +1,184 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.security.keyvault.jca;
+
+import com.azure.security.keyvault.jca.rest.OAuthToken;
+
+import java.util.HashMap;
+import java.util.logging.Logger;
+
+import static java.util.logging.Level.FINER;
```

Comment:
It looks like elsewhere you have a dependency on slf4j - it would be good to standardise on that (which is also what the azure SDKs have standardised on).

Reply:
As the JAR is self-contained it is not relying on SLF4J for its logging. Even though Jackson is pulling it in as a dependency it is not used for logging by the JCA provider itself.